components/dashboard/src/settings/Teams.tsx

```diff
@@ -0,0 +1,846 @@
+                )}
+                {getActiveSubs().map((sub, index) => (
+                    <div key={"team-sub-" + sub.id} className="flex-grow flex flex-row hover:bg-gray-100 rounded-xl h-16 w-full group">
+                        <div className="px-4 self-center w-1/12">
+                            <div className={"rounded-full w-3 h-3 text-sm align-middle " + (isPaymentInProgress(sub) ? "bg-gitpod-kumquat" : "bg-green-500") }>
+                                &nbsp;
+                        </div>
+                        </div>
+                        <div className="p-0 my-auto flex flex-col w-4/12">
+                            <span className="my-auto font-medium truncate overflow-ellipsis">{getPlan(sub)?.name}</span>
+                            <span className="text-sm my-auto text-gray-400 truncate overflow-ellipsis">Purchased on {formatDate(sub?.startDate)}</span>
+                        </div>
+                        <div className="p-0 my-auto flex flex-col w-2/12">
+                            <span className="my-auto truncate text-gray-500 overflow-ellipsis">{slots.filter(s => s.teamSubscription.id === sub.id).length || "–"}</span>
+                            <span className="text-sm my-auto text-gray-400">Members</span>
+                        </div>
+                        <div className="p-0 my-auto flex w-4/12">
+                            {isPaymentInProgress(sub) && (
+                                <div className="ml-auto self-center rounded-xl bg-orange-300 font-medium text-sm text-orange-700 py-1 px-2">
+                                    Payment in Progress
+                                </div>
+                            )}
+                        </div>
+                        <div className="my-auto flex w-1/12 pl-8 opacity-0 group-hover:opacity-100">
+                            <div className="self-center hover:bg-gray-200 rounded-md cursor-pointer w-8">
+                                <ContextMenu menuEntries={subscriptionMenu(sub)}>
+                                    <img className="w-8 h-8 p-1" src={ThreeDots} alt="Actions" />
+                                </ContextMenu>
+                            </div>
+                        </div>
+                    </div>
+                ))}
+            </div>
+        )}
+    </React.Fragment>);
+
+    return (<div>
+        {showPaymentUI ? renderTeams() : (
+            <div className="flex flex-row">
+                <div className="flex-grow ">
+                    <h3 className="self-center">All Teams</h3>
+                    <h2>Manage teams and team members.</h2>
+                </div>
+            </div>
+        )}
+    </div>);
+}
+
+function InviteMembersModal(props: {
+    sub: TeamSubscription,
+    onClose: () => void
+}) {
+
+    const [copied, setCopied] = useState<boolean>(false);
+
+    const getInviteURL = () => {
+        const link = new URL(window.location.href);
+        link.pathname = '/subscription/'
```

Comment:
Well, this will still actually not work, but I think we'll implement this in the `/plans` page soon:

```suggestion
        link.pathname = '/plans'
```